doc/source/whatsnew/v1.2.0.rst

```diff
@@ -729,6 +729,7 @@ I/O
 - :meth:`read_fwf` was inferring compression with ``compression=None`` which was not consistent with the other :meth:``read_*`` functions (:issue:`37909`)
```

Comment:
i think italics are messed up bc the "read*" should have single-ticks instead of double-ticks

Reply:
Changed it